sled-agent/src/zone_bundle.rs

```diff
@@ -1545,61 +1550,29 @@ async fn compute_bundle_utilization(
 
 // Return the number of bytes occupied by the provided directory.
 //
-// This returns an error if:
-//
-// - The "du" command fails
-// - Parsing stdout fails
-// - Parsing the actual size as a u64 fails
+// This returns an error if reading any file or directory within the provided
+// directory fails.
 async fn disk_usage(path: &Utf8PathBuf) -> Result<u64, BundleError> {
```

Comment:
I think it's probably worth considering a different name here, given the function appears to be assessing the apparent size of a tree of files rather than the actual disk usage.  I definitely would have guessed the other way based on the name!

Reply:
Ugh, that's a good point --- `du(1)` was actually reporting physical disk usage. We should probably change this to be more faithful to `du` rather than reporting logical size...

Reply:
We probably  want [`std::fs::unix::MetadataExt::blocks`](https://doc.rust-lang.org/stable/std/os/unix/fs/trait.MetadataExt.html#tymethod.blocks)?

Reply:
> Ugh, that's a good point --- `du(1)` was actually reporting physical disk usage. 

I don't believe it was, because we were using the `-A` flag.  To be clear I think the name was already confusing haha

Reply:
Ahh, okay, I can just rename the function, then. Unless we _want_ the other behavior --- I'm not totally sure. @bnaecker, this is your code, right?

Reply:
It is, yeah. I have no preference on the name, that was chosen since it was calling out to `du`. Whatever you want to call it is fine by me!

Reply:
772cd163c523f21a67ea543ccf55e03ef6e14d5d renames it to `dir_size`, but I'm happy to change the name again if there's something y'all think would be clearer.

Reply:
Can that be &Utf8Path btw? I know you didn't touch that specific line

Reply:
It can, yeah — I just didn't want to mess with the surrounding code as part of a quick fix. Happy to go and change it as well.